### PR TITLE
build: fix version number

### PR DESCRIPTION
**Pull request by cpcloud**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "koerce"
-version = "0.0.1"
+version = "0.0.7"
```
Comment (kszucs) [inline pyproject.toml:3]: I tried following the poetry-dynamic-versioning guide: https://pypi.org/project/poetry-dynamic-versioning/

> Poetry still requires the tool.poetry.version field to be present in pyproject.toml, but you are encouraged to use version = "0.0.0" as a standard placeholder.

Could we use `0.0.0` here?

Comment (cpcloud) [reply pyproject.toml:3]: Oh, yeah, probably.

I didn't really pay attention to whether dynamic verisoning was implemented and was just being lazy.

Comment (kszucs) [reply pyproject.toml:3]: Can you update to 0.0.0 then I would merge it.

Comment (cpcloud) [reply pyproject.toml]: Done.

**Review by kszucs — APPROVED** [review]
Thanks!

**Pull request merged**